lib/router.js

```diff
@@ -36,10 +36,15 @@ Cherrytree.prototype.initialize = function (options) {
   this.middleware = []
   this.options = _.extend({
     interceptLinks: true,
-    logError: true
+    logError: true,
+    Promise: window.Promise
```

Comment:
need to do smth like `typeof window === 'undefined' ? global.Promise : window.Promise` to continue supporting node env.